src/state.rs

```diff
@@ -247,6 +254,16 @@ impl NetworkState {
         Ok(ret)
     }
 
+    /// Returns the root of the tree in the `head` state.
+    pub fn root(&self) -> [u8; 32] {
+        self.head.root()
+    }
+
+    /// Returns the root of the tree in the `origin` state.
+    pub fn origin_root(&self) -> [u8; 32] {
+        self.origin.root()
+    }
```

Comment:
Could you double check with @toghrulmaharramov,or  @vlopes11 and  @goshawk-3 if we ever need the origin's root? Since at first glance I don't think we need it, but I might be mistaken. And even if we discover later than we need it, it's easier to add a public API (minor change) than removes one (breaking change).

Reply:
Just took a look at the schemas and it doesn´t seem to me like we do either. Will remove.

Reply:
Done